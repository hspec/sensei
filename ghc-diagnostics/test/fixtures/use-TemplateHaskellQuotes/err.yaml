version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/use-TemplateHaskellQuotes/Foo.hs
  start:
    line: 2
    column: 7
  end:
    line: 2
    column: 13
severity: Error
code: 62558
message:
- Syntax error on [| 23 |]
- In the Template Haskell quotation [| 23 |]
hints:
- |-
  Perhaps you intended to use the `TemplateHaskellQuotes' extension (implied by `TemplateHaskell')
  You may enable this language extension in GHCi with:
    :set -XTemplateHaskellQuotes
