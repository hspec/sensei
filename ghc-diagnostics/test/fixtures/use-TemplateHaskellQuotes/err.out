test/fixtures/use-TemplateHaskellQuotes/Foo.hs:2:7: error: [GHC-62558]
    • Syntax error on [| 23 |]
    • In the Template Haskell quotation [| 23 |]
    Suggested fix:
      Perhaps you intended to use the `TemplateHaskellQuotes' extension (implied by `TemplateHaskell')

